iree/modules/strings/BUILD

```diff
@@ -31,6 +31,13 @@ cc_library(
     ],
 )
 
+iree_cmake_extra_content(
```

Comment:
errors from the Bazel CI run: https://source.cloud.google.com/results/invocations/76281b45-dc2f-4f08-9f19-85b53df423af/targets/iree%2Fgcp_ubuntu%2Fbazel%2Flinux%2Fx86-swiftshader%2Fcore%2Fpresubmit/log

```
ERROR: /tmpfs/src/github/iree/iree/modules/strings/BUILD:34:1: name 'iree_cmake_extra_content' is not defined
ERROR: /tmpfs/src/github/iree/iree/vm/BUILD:215:1: name 'iree_cmake_extra_content' is not defined
ERROR: /tmpfs/src/github/iree/iree/vm/BUILD:281:1: name 'iree_cmake_extra_content' is not defined
ERROR: /tmpfs/src/github/iree/iree/vm/test/BUILD:24:1: name 'iree_cmake_extra_content' is not defined
ERROR: /tmpfs/src/github/iree/iree/modules/tensorlist/BUILD:23:1: name 'iree_cmake_extra_content' is not defined
ERROR: /tmpfs/src/github/iree/iree/vm/test/BUILD:77:1: name 'iree_cmake_extra_content' is not defined
ERROR: /tmpfs/src/github/iree/iree/modules/strings/BUILD:70:1: name 'iree_cmake_extra_content' is not defined
ERROR: /tmpfs/src/github/iree/iree/modules/tensorlist/BUILD:60:1: name 'iree_cmake_extra_content' is not defined
ERROR: package contains errors: iree/modules/strings
ERROR: package contains errors: iree/vm/test
ERROR: package contains errors: iree/modules/tensorlist
ERROR: package contains errors: iree/vm
```

make sure 
```
load("//iree:build_defs.oss.bzl", "iree_cmake_extra_content")
```
is included in those files (and buildifier is happy with the formatting)

Reply:
Fixed!